spec/models/application_search_filter_spec.rb

```diff
@@ -49,16 +49,16 @@
     end
   end
 
-  describe '#to_json' do
-    subject(:to_json) { new.to_json }
+  describe '#datastore_params' do
+    subject(:datastore_params) { new.datastore_params }
 
     context 'when the filter is empty' do
-      it 'returns the correct datastore api search params as json' do
-        expect(to_json).to eq({
-          applicant_date_of_birth: nil, application_id_in: [],
+      it 'returns the correct datastore api search params' do
+        expect(datastore_params).to eq({
+                                         applicant_date_of_birth: nil, application_id_in: [],
             application_id_not_in: [], status: ['submitted'],
             submitted_after: nil, submitted_before: nil, search_text: nil
-        }.to_json)
+                                       })
```

Comment:
Is the spacing of the brackets here a little off? I would expect it to align with "expect"